apps/wallet-ui/src/lib/ecosystems.ts

```diff
@@ -0,0 +1,23 @@
+import type { Ecosystem } from "@/types/Ecosystem";
+
+export async function getEcosystemInfo(
+  ecosystemId: string,
+): Promise<Ecosystem> {
+  const response = await fetch(
+    "https://embedded-wallet.thirdweb.com/api/2024-05-05/ecosystem-wallet",
+    {
+      headers: {
+        "x-ecosystem-id": `ecosystem.${ecosystemId}`,
+      },
```

Comment:
I assume this would need some kind of auth also?

Reply:
It does not (if that's a problem we'll need to fix it in ews)

Reply:
I think it was by design though since you _in theory_ shouldn't need a client ID to use ecosystem wallets (not the reality today)